projects/plugins/social/src/class-note.php

```diff
@@ -107,25 +109,42 @@ public function register_cpt() {
 				'items_list_navigation' => esc_html__( 'Notes list navigation', 'jetpack-social' ),
 				'items_list'            => esc_html__( 'Notes list', 'jetpack-social' ),
 			),
-			'show_in_rest'  => true,
-			'has_archive'   => true,
-			'supports'      => array( 'editor', 'comments', 'thumbnail', 'publicize', 'enhanced_post_list', 'activitypub' ),
-			'menu_icon'     => 'dashicons-welcome-write-blog',
-			'rewrite'       => array( 'slug' => 'sn' ),
-			'template'      => array(
+			'show_in_rest' => true,
+			'has_archive'  => true,
+			'supports'     => array( 'editor', 'comments', 'thumbnail', 'publicize', 'enhanced_post_list', 'activitypub' ),
+			'menu_icon'    => 'dashicons-welcome-write-blog',
+			'rewrite'      => array( 'slug' => 'sn' ),
+			'template'     => array(
 				array(
 					'core/paragraph',
 					array(
 						'placeholder' => __( "What's on your mind?", 'jetpack-social' ),
 					),
 				),
 			),
-			'template_lock' => 'all',
 		);
 		register_post_type( self::JETPACK_SOCIAL_NOTE_CPT, $args );
 		self::maybe_flush_rewrite_rules();
 	}
 
+	/**
+	 * Restrict the blocks for the Social Note CPT.
+	 *
+	 * @param array $allowed_blocks The allowed blocks.
+	 * @param array $post The post.
```

Comment:
I think phan is complaining as $post is declared as an array. I think it can be a `WP_Post`